torch/csrc/jit/python/script_init.cpp

```diff
@@ -1339,6 +1341,29 @@ void initJitScriptBindings(PyObject* module) {
 
         const auto self = SimpleSelf(classType);
         cu->define(classname, props, propRcbs, methodDefs, methodRcbs, &self);
+
+        // Stitch in default arguments for methods. Properties don't need to be
+        // considered since there is no way to invoke setters without passing in
+        // a value.
+        auto defs_it = methodDefs.begin();
+        while (defs_it != methodDefs.end()) {
+          auto def_name = (*defs_it).name().name();
+          // If the method is not in the defaults map, assume there are
+          // no default arguments for it.
+          if (defaults.find(def_name) == defaults.end()) {
```

Comment:
nit:

save results of `defaults.find(def_name)` and reuse on line 1364 to look up only once

Reply:
Nice catch.